src/promptflow/tests/executor/unittests/processpool/test_line_execution_process_pool.py

```diff
@@ -150,11 +147,7 @@ def execute_in_spawn_mode_subprocess(
 
 
 def create_line_execution_process_pool(dev_connections):
-    executor = FlowExecutor.create(
-        get_yaml_file(SAMPLE_FLOW),
-        dev_connections,
-        line_timeout_sec=1,
-    )
+    executor = FlowExecutor.create(get_yaml_file(SAMPLE_FLOW), dev_connections)
```

Comment:
We may change to use other flows instead of sample flow, maybe in another PR.

Reply:
ok